public/media-hunt/index.js

```diff
@@ -1 +1,27 @@
-console.log("Media hunt");
\ No newline at end of file
+const args = process.argv.slice(2);
+
+const commands = {
+  configuration: () => require("./commands/configuration").run(),
+  movies: () => require("./commands/movies/movies").run(),
+  movie: (id) => require("./commands/movies/movie").run(id),
+  movieGanres: () => require("./commands/movies/ganres").run(),
+  movieReviewes: (id) => require("./commands/movies/reviews").run(id),
+  movieTopRated: () => require("./commands/movies/toprated").run(),
+  TVs: () => require("./commands/tv/tvs").run(),

```

Comment:
let's stick to [camelcase](https://khalilstemmler.com/blogs/camel-case-snake-case-pascal-case/) for command declarations.